scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl

```diff
@@ -47,7 +49,7 @@ workflow GvsCreateVATFilesFromBigQuery {
             contig_array = contig_array,
             output_path = output_path,
             merge_vcfs_disk_size_override = merge_vcfs_disk_size_override,
-            cloud_sdk_docker = effective_cloud_sdk_docker,
+            cloud_sdk_docker = effective_cloud_sdk_slim_docker,
```

Comment:
input name should also be `slim`

Reply:
Okay - resolved this and the parameterization of `GvsCreateVATFromVDS.wdl`